README.md

```diff
@@ -1,32 +1,235 @@
+
+To verify the integrity of the binaries on macOS run the following:
 
-### Development
+```sh
+$ shasum -c checksums.txt --ignore-missing
+drs-downloader-macOS.zip: OK
+```
+
+If the `shasum` command outputs `OK` than the verification was successful. 
+
+#### Unsuccessful Verification
+
+Alternatively if the commad outputs `FAILED` than the checksum did not match and the binary should not be run. In such a case please reach out to the contributors for assistance.
+
+```sh
+$ shasum -c checksums.txt --ignore-missing
+drs-downloader-macOS.zip: FAILED
+shasum: WARNING: 1 computed checksum did NOT match
+shasum: checksums.txt: no file was verified
+```
+
+</details>
 
 ## Usage
 
+### Quick Start
+
+```sh
+drs_downloader terra -m <manifest file> -d <destination directory>
+```
+
+### Example
+
+The below command is a basic example of how to structure a download command with all of the required arguments:
+
+```sh
+$ drs_downloader terra -m tests/fixtures/terra-data.tsv -d DATA
+100%|████████████████████████████████| 10/10 [00:00<00:00, 56148.65it/s]
+
+2022-11-21 16:56:49,595 ('HG03873.final.cram.crai', 'OK', 1351946, 1)
+2022-11-21 16:56:49,595 ('HG04209.final.cram.crai', 'OK', 1338980, 1)
+2022-11-21 16:56:49,595 ('HG02142.final.cram.crai', 'OK', 1405543, 1)
+2022-11-21 16:56:49,595 ('HG01552.final.cram.crai', 'OK', 1296198, 1)
+2022-11-21 16:56:49,595 ('NA18613.final.cram.crai', 'OK', 1370106, 1)
+2022-11-21 16:56:49,595 ('HG00536.final.cram.crai', 'OK', 1244278, 1)
+2022-11-21 16:56:49,595 ('HG02450.final.cram.crai', 'OK', 1405458, 1)
+2022-11-21 16:56:49,595 ('NA20525.final.cram.crai', 'OK', 1337382, 1)
+2022-11-21 16:56:49,595 ('NA20356.final.cram.crai', 'OK', 1368064, 1)
+2022-11-21 16:56:49,595 ('HG00622.final.cram.crai', 'OK', 1254920, 1)
+2022-11-21 16:56:49,595 ('done', 'statistics.max_files_open', 37) 
+
+$ ls ./DATA
+HG00536.final.cram.crai HG01552.final.cram.crai
+HG02450.final.cram.crai HG04209.final.cram.crai
+NA20356.final.cram.crai HG00622.final.cram.crai
+HG02142.final.cram.crai HG03873.final.cram.crai
+NA18613.final.cram.crai NA20525.final.cram.crai
+```
+
+### Additional Options
+
+To see all available flags run the `help` command:
+
+```sh
+drs_downloader terra --help
+```
+
+```sh
+Usage: drs_download terra [OPTIONS]
+
+  Copy files from terra.bio
+
+Options:
+  -s, --silent                Display nothing.
+  -d, --destination_dir TEXT  Destination directory.  [default: /tmp/testing]
+  -m, --manifest_path TEXT    Path to manifest tsv.  [default:
+                              tests/fixtures/terra-data.tsv]
+  --help                      Show this message and exit.
+```
+## Development
+
+To get ready for development first get the code:
+
+```sh
+git clone https://github.com/anvilproject/drs_downloader
+cd drs_downloader
+```
+
+Then create and activate a virtual environment using `Python3.9`:
+
+```sh
+python3.9 -m venv venv
+. venv/bin/activate
+pip install -r requirements.txt -r requirements-dev.txt
+```
+
```

Comment:
I think a step is missing here.
After running the commands above then running 
```
pytest --cov=tests
```
pytest reported many errors.

I then ran the following from the top-level repo directory:
```
pip install -e .
```
then pytest ran much better.

I am not sure if `pip install -e .` is the right/best command to run here, yet it seems like there is a missing step in the insturctions.

Reply:
Agreed.  Will address today in a change to this PR, (change to README)

Reply:
[Fixed](https://github.com/anvilproject/drs_downloader/blob/feature/manager/README.md#development)